test/topology/test_change_ip.py

```diff
@@ -28,3 +29,30 @@ async def test_change_two(manager, random_tables):
     await manager.server_start(s_2)
     await table.add_column()
     await random_tables.verify_schema()
+
+
+@pytest.mark.asyncio
+async def test_change_all(manager, random_tables):
+    """Add a node, stop an original node, add a column"""
+    servers = await manager.running_servers()
+    table = await random_tables.add_table(ncolumns=5)
+    stops = []
+    for server in servers:
+        logger.info("Gracefully stopping server %s to change ip", server.server_id)
+        stops.append(manager.server_stop_gracefully(server.server_id))
+    await asyncio.gather(*stops)
+
+    starts = []
+    for server in servers:
+        logger.info("Starting server %s", server.server_id)
+        await manager.server_change_ip(server.server_id)
+        starts.append(manager.server_start(server.server_id))
+    await asyncio.gather(*starts)
+    # Despite hacks in server_start() to refresh the connection list, if
+    # we shut down the entire cluster, they don't work. TODO: investigate
+    # why. Meanwhile, without a manual close/connect the test tear down
```

Comment:
The driver may not be able to reconnect after we change all IPs.

Perhaps we need to return the new IPs from `server_change_ip` (which is a good idea anyway, we can use it for logging etc.) and then reconnect the driver.

The Manager server-side driver  connection should probably be reestablished with new IPs as well...

Reply:
No longer relevant.